spec/spec_helper.rb

```diff
@@ -25,25 +25,29 @@ def create_page_with_sourcebox iUrl=nil, subcards={},sourcebox=nil
   end
 end
 
-def create_claim_with_url name,url, subcards={}
+def create_claim_with_url name, url, subcards={}
   Card::Auth.as_bot do
     sourcepage = create_page url
-    Card.create! :type_id=>Card::ClaimID, :name=>name, 
-                 :subcards=>{ '+source' => {:content=>"[[#{sourcepage.name}]]",:type_id=>Card::PointerID}}.merge(subcards)    
+    Card.create! type_id: Card::ClaimID, :name=>name,
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator `=>`.